.env.template

```diff
@@ -1,11 +1,12 @@
 PORT=5001
+CACHE_WARMING_PORT=8080
 URL=http://localhost:5001
 BACKEND_PATH=/api
 GRAPHQL_PATH=/graphql
 NODE_ENV=development
 MONGODB_URI=mongodb://mongodb:27017/bt?replicaSet=rs0
 REDIS_URI=redis://redis:6379
-BACKEND_URL=http://localhost:8080
+BACKEND_URL=http://backend:8080
```

Comment:
If you're separating domain and port, you should remove port from `BACKEND_URL`. You would also need to update `infra/app/templates/datapuller.yaml`.

I think separating is unnecessary though since they should always be used together.